<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_dictionary</class>
 <widget class="QMainWindow" name="window_dictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word Sea</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/uil--book-reader.png</normaloff>icons/uil--book-reader.png</iconset>
  </property>
  <widget class="QWidget" name="widget_main">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="layout_input">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="line_input">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter a vocabulary</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_search">
        <property name="text">
         <string>Search</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_clear">
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_remove">
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer_horizontal">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_learn">
        <property name="text">
         <string>Learn</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="list_cluster">
      <property name="spacing">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_reset"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menu_theme">
     <property name="title">
      <string>Theme</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/uil--brush-alt.png</normaloff>icons/uil--brush-alt.png</iconset>
     </property>
     <addaction name="action_theme_default"/>
     <addaction name="action_theme_dark"/>
     <addaction name="separator"/>
     <addaction name="action_import_qss"/>
    </widget>
    <addaction name="menu_theme"/>
    <addaction name="action_list"/>
    <addaction name="action_flashcard"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_view"/>
  </widget>
  <action name="action_add">
   <property name="text">
    <string>Add Vocabulary</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="icon">
    <iconset>
     <normaloff>icons/bxs--save.png</normaloff>icons/bxs--save.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset>
     <normaloff>icons/bx--save.png</normaloff>icons/bx--save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset>
     <normaloff>icons/uil--folder-open.png</normaloff>icons/uil--folder-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionsView">
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="action_list">
   <property name="icon">
    <iconset>
     <normaloff>icons/uil--list-ul.png</normaloff>icons/uil--list-ul.png</iconset>
   </property>
   <property name="text">
    <string>List</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="icon">
    <iconset>
     <normaloff>icons/uil--trash-alt.png</normaloff>icons/uil--trash-alt.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="action_theme_default">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="action_theme_dark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="action_flashcard">
   <property name="icon">
    <iconset>
     <normaloff>icons/mingcute--board-line.png</normaloff>icons/mingcute--board-line.png</iconset>
   </property>
   <property name="text">
    <string>Flashcard</string>
   </property>
  </action>
  <action name="action_import_qss">
   <property name="icon">
    <iconset>
     <normaloff>icons/uil--import.png</normaloff>icons/uil--import.png</iconset>
   </property>
   <property name="text">
    <string>Import QSS</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
